thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/controller.js

```diff
@@ -248,6 +248,23 @@ export default Ember.Controller.extend({
       }
     },
 
+    /**
+     * Send a DELETE request to the function delete endpoint.
+     * @method removeThirdEyeFunction
+     * @param {Number} functionId - The id of the alert to remove
+     * @return {Promise}
+     */
+    removeThirdEyeFunction(functionId) {
+      const postProps = {
```

Comment:
nit: Let's move this into a `/utils/constant.js` since this is needed for every post request with fetch.

Reply:
My bad. Its actually already there in utils. Forgot to use it.

Reply:
On second thought, I do have such a utility, but needs refactoring - its only for POST requests. I'll make a note to include it.